Makefile

```diff
@@ -47,12 +46,10 @@ generate:
 	$(GO) generate
 
 build: generate
-	$(GO) build -o $(BIN) -ldflags $(LDFLAGS)
+	CGO_ENABLED=0 $(GO) build -o $(BIN) -ldflags $(LDFLAGS)
```

Comment:
Disabling CGO is probably fine for now. If we need it in the future, we can turn it back on.